tools/docker/bazel.dockerfile

```diff
@@ -6,6 +6,11 @@ ARG TARGETARCH
 COPY --chown=0:0 bazel/install-deps.sh /
 
 RUN dnf install -y wget || { apt-get update && apt install -y wget; }
+RUN dnf install -y curl || { apt-get update && apt install -y curl; }
+RUN dnf install -y pigz || { apt-get update && apt install -y pigz; }
+RUN dnf install -y patchelf || { apt-get update && apt install -y patchelf; }
+RUN dnf install -y devscripts || { apt-get update && apt install -y devscripts; }
+RUN dnf install -y debhelper || { apt-get update && apt install -y debhelper; }
```

Comment:
Better to add all the package names in one line and not make a bunch of repeated pointless calls to `apt-get update`

Reply:
done